sources/reference/admiral-aws-marketplace.md

```diff
@@ -7,100 +7,120 @@ page_keywords: install, microservices, Continuous Integration, Continuous Deploy
 
 # Installing from AWS Marketplace
 
-Shippable Server - Startup Edition (SE) is available on AWS Marketplace and can be installed by going
-through a one-click purchase and install process. This will bring up Shippable Server on an EC2
-instance of desired configuration. Once launched, you will configure the installation
-to connect your third party services (like your source control system) using the Admin panel.
+Shippable Server - Startup Edition (SE) is available on the AWS Marketplace to
+run on a single instance, ideal for evaluations/POCs or for small teams.  Once
+launched, you will configure the installation to connect your third party
+services (like your source control system) using the Admin panel.
 
-## Purchase process
-When purchasing Shippable Server SE from the AWS Marketplace, you'll make the
-purchase via your AWS account using the payment method associated with your account. All
-Shippable Server SE purchases made through the AWS Marketplace come with a 30-day free trial
-and have the option for hourly or annual billing, with annual billing provided at a
-discount.
+## Purchase and install process 
+Purchase and install Shippable Server SE by visiting [its listing on the AWS
+Marketplace](add link). When purchasing Shippable Server SE from the AWS
+Marketplace, you'll make the purchase via your AWS account using the payment
+method associated with your account. 
 
-You will be notified by AWS prior to completion of your 30-day free trial with the option to
-cancel your purchase prior to completing the trial period.
+All Shippable Server SE purchases made through the AWS Marketplace come with a
+**30-day free trial** with the option for hourly or annual billing. Annual 
+billing is provided at a discount to the hourly rate. 
 
-If annual billing selected, you will be notified by AWS prior to your annual term
-expiring with the option to extend for another annual term. If you choose not to renew, upon
-completing the annual term, billing will automatically convert to hourly billing.
+You will be notified by AWS prior to completion of your 30-day free trial with
+the option to cancel your purchase prior to completing the trial period.
```

Comment:
Maybe remove "prior to completing the trial period."  I don't think "prior to" needs to be in the same sentence twice.  Or find a different phrase for one of the "prior to"s.